marconi-chain-index/src/Marconi/ChainIndex/Experimental/Indexers/MintTokenEvent.hs

```diff
@@ -433,27 +436,34 @@ newtype MintTokenEventsMatchingQuery event = MintTokenEventsMatchingQuery (event
 
 type instance
   Core.Result (MintTokenEventsMatchingQuery event) =
-    [Core.Timed (Core.Point event) event]
+    [Core.Stability (Core.Timed C.ChainPoint event)]
 
 instance
-  (MonadIO m, MonadError (Core.QueryError (MintTokenEventsMatchingQuery MintTokenBlockEvents)) m)
+  ( MonadIO m
+  , MonadError (Core.QueryError (MintTokenEventsMatchingQuery MintTokenBlockEvents)) m
+  )
   => Core.Queryable
       m
       MintTokenBlockEvents
       (MintTokenEventsMatchingQuery MintTokenBlockEvents)
       Core.SQLiteIndexer
   where
-  query =
+  query point query indexer =
```

Comment:
Genuine question: does it improve your readability to expand the parameter (I tend to eta-reduce to notify that the parameters aren't required in the function body, but it may be a personal bias).

Reply:
Oh my bad, I didn't spot the use of `indexer` below. Thanks.
(and I don't think we have a specific guideline for eta-reduction, which is actually why I asked)